packages/@tailwindcss-upgrade/src/index.test.ts

```diff
@@ -175,6 +243,38 @@ it('should migrate a stylesheet (with preceding rules that should be wrapped in
     /**! My license comment */
     @import 'tailwindcss';
     @config './tailwind.config.js';
+    /*
```

Comment:
Unsure why there is no empty line above this one here?

Reply:
Fun part is that the space between the `@import` and `@config` is also gone while it should be there. 

So I feel like whitespace handling might need a dedicated PR to get it all right. But I'll take a look at the other cases to see if it's easily fixable or not.

Bigger problem is that in isolation it's correct, but the accumulation of changes is not.

Reply:
Figured it out. In this case the test is performing migrations and once done formats the nodes on a stringified version. The actual integration tests don't have this issue.